tests/pet.test.js

```diff
@@ -109,4 +113,15 @@ describe('checks if pet is alive',() => {
     pet.hunger = 9;
```

Comment:
In the first test, to check if the pet is alive, you could only set the age to 30, fitness to 0 or hunger to 10. If your code is right, as it seems to be, it will triger the function's value to be falsy. No need to use all 3 conditions. What you could do is test each of them separately and check if it validates the same logic.